seed_identity_store/settings.py

```diff
@@ -150,7 +162,7 @@
 CELERY_CREATE_MISSING_QUEUES = True
 
 CELERY_TASK_SERIALIZER = 'json'
-CELERY_RESULT_SERIALIZER = 'json'
+CELERY_RESULT_SERIALIZER = 'pickle'
```

Comment:
@hodgestar unfortunately yes, rest_hooks needs it. @imsickofmaps has worked with the project before and said he'd see if he can get them to fix it.